src/app/modules/main/profile_section/contacts/controller.nim

```diff
@@ -16,9 +17,6 @@ type
     contactsService: contacts_service.Service
     accountsService: accounts_service.ServiceInterface
 
-# forward declaration:
-method getContacts*[T](self: Controller[T]): seq[ContactsDto]
```

Comment:
Any reason this was removed? I assume this was introduced for a reason as well...